flexget/utils/template.py

```diff
@@ -136,6 +137,26 @@ def filter_default(value, default_value: str = '', boolean: bool = True) -> str:
     return jinja2.filters.do_default(value, default_value, boolean)
 
 
+def filter_asciify(text: str) -> str:
+    """ Siplify text """
+
+    if not isinstance(text, str):
+        return text
+
+    # Replace accented chars by their 'normal' couterparts
+    result = normalize('NFKD', text)
+
+    # Symbols that should keeped as white space
+    result = re.sub(r'[ \(\)\-_\[\]\.]+', ' ', result)
+    # Leftovers
+    result = re.sub(r"[^a-zA-Z0-9 ]", "", result)
+
+    # Remove Multiple White Spaces to One
+    result = ' '.join(result.split())
```

Comment:
This will replace all whitespace with a single space. Probably the desired result, but wanted to make sure it was intentional.

Reply:
That's correct! And also remove all leading and trailing white spaces!

This should be done because the replaces can leave multiple white spaces if more that one special char is together